src/js/components/form/FormGroup.js

```diff
@@ -5,14 +5,14 @@ import FieldError from './FieldError';
 import {omit} from '../../utils/Util';
 
 const FormGroup = (props) => {
-  const {children, className, errorClassName, showError, required} = props;
+  const {children, className, errorClassName, showError} = props;
 
   const clonedChildren = React.Children.map(children, (child) => {
-    if (child == null || (!showError && child.type === FieldError)) {
```

Comment:
so we're changing the logic here with no replacement, meaning that if we merge it we break existing functionality?